deploy/service.yaml

```diff
@@ -17,15 +17,15 @@ spec:
     spec:
       containers:
         - name: ansibletower-service
-          image: gabrielprioli/ansibletower-service:latest # Todo: Replace this with your image name
+          image: jetzlstorfer/ansibletower-service:dev # Todo: Replace this with your image name
```

Comment:
@jetzlstorfer do you want to reference the image in your account here?

Reply:
That was a bit by accident, will change it to `keptnsandbox` as we want to push the images there

Reply:
ok

Reply:
fixed now